src/main/java/machines/Bulldozer.java

```diff
@@ -0,0 +1,11 @@
+package machines;
+
+public class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Bulldozer has started the job");
```

Comment:
The message should be more informative and indicate the specific type of machine that is working. Consider including the class name in the message.